articles/aks/load-balancer-standard.md

```diff
@@ -101,9 +101,9 @@ Labels:                      <none>
 Annotations:                 <none>
 Selector:                    app=public-app
 ...
-IP:                          10.0.39.110
+IP:                          xx.x.xx.xxx
 ...
-LoadBalancer Ingress:        52.156.88.187
+LoadBalancer Ingress:        xx.xxx.xx.xxx
```

Comment:
```suggestion
LoadBalancer Ingress:        203.0.113.187
```

Reply:
We don't want IPs replaced with xxx's.  The cluster IP is in the private range and is approved.  The public IP is in the approved range for documentation.  
https://review.learn.microsoft.com/en-us/help/contribute/contribute-legal-guidelines?branch=main#ip-addresses